tests/test_decoder_models.py

```diff
@@ -157,6 +157,19 @@ def test_from_pretrained_config_construction(self, test_name, model_id, config_c
     def test_merge_layers(self, test_name, model_id, config_cls, config_kwargs):
         self._test_merge_layers(model_id, config_cls, config_kwargs)
 
+    @parameterized.expand(
+        PeftTestConfigManager.get_grid_parameters(
+            {
+                "model_ids": PEFT_DECODER_MODELS_TO_TEST,
+                "lora_kwargs": {"init_lora_weights": [False]},
+                "ia3_kwargs": {"init_ia3_weights": [False]},
+                "task_type": "CAUSAL_LM",
+            },
+        )
+    )
+    def test_merge_layers_multi(self, test_name, model_id, config_cls, config_kwargs):
```

Comment:
I don't think we need these tests for all model types besides decoder. But maybe let's also add them to the custom model tests, as we check a broader range of adapters there.